### ISSUE TITLE
`extract_word': undefined method `sort!' for ...

### ISSUE THREAD

**Issue by fnorf** [tsdh/rdictcc#3]
Just discovered and installed this awesome thing!
Sadly importing the de-en and en-de files fails for me. I guess it is some encoding problem?

My version is r36.43192fc-1 installed from https://aur.archlinux.org/packages/rdictcc-git/

```
** Reading dict file (langA)
Traceback (most recent call last):
	13: from /usr/bin/rdictcc:432:in `<main>'
	12: from /usr/lib/ruby/2.6.0/optparse.rb:1678:in `parse!'
	11: from /usr/lib/ruby/2.6.0/optparse.rb:1656:in `permute!'
	10: from /usr/lib/ruby/2.6.0/optparse.rb:1562:in `order!'
	 9: from /usr/lib/ruby/2.6.0/optparse.rb:1568:in `parse_in_order'
	 8: from /usr/lib/ruby/2.6.0/optparse.rb:1568:in `catch'
	 7: from /usr/lib/ruby/2.6.0/optparse.rb:1582:in `block in parse_in_order'
	 6: from /usr/bin/rdictcc:383:in `block (2 levels) in <main>'
	 5: from /usr/bin/rdictcc:144:in `import'
	 4: from /usr/bin/rdictcc:194:in `read_dict_file'
	 3: from /usr/bin/rdictcc:194:in `foreach'
	 2: from /usr/bin/rdictcc:200:in `block in read_dict_file'
	 1: from /usr/bin/rdictcc:219:in `add_line'
/usr/bin/rdictcc:241:in `extract_word': undefined method `sort!' for "$500 suit":String (NoMethodError)
```

```
** Reading dict file (langA)
Traceback (most recent call last):
	13: from /usr/bin/rdictcc:432:in `<main>'
	12: from /usr/lib/ruby/2.6.0/optparse.rb:1678:in `parse!'
	11: from /usr/lib/ruby/2.6.0/optparse.rb:1656:in `permute!'
	10: from /usr/lib/ruby/2.6.0/optparse.rb:1562:in `order!'
	 9: from /usr/lib/ruby/2.6.0/optparse.rb:1568:in `parse_in_order'
	 8: from /usr/lib/ruby/2.6.0/optparse.rb:1568:in `catch'
	 7: from /usr/lib/ruby/2.6.0/optparse.rb:1582:in `block in parse_in_order'
	 6: from /usr/bin/rdictcc:383:in `block (2 levels) in <main>'
	 5: from /usr/bin/rdictcc:144:in `import'
	 4: from /usr/bin/rdictcc:194:in `read_dict_file'
	 3: from /usr/bin/rdictcc:194:in `foreach'
	 2: from /usr/bin/rdictcc:200:in `block in read_dict_file'
	 1: from /usr/bin/rdictcc:219:in `add_line'
/usr/bin/rdictcc:241:in `extract_word': undefined method `sort!' for "&#945; keratin":String (NoMethodError)

```

**Comment by tsdh** [comment]
Oh, the return type of `String::split` when given a block changed in a recent ruby version!

I'll fix that this evening (hopefully)!

**Comment by tsdh** [comment]
@fnorf Should be fixed by 8d3962c90e1e3d45bf90c5e2f3932218274dec3a.

Thanks for reporting!

**Comment by fnorf** [comment]
Lots of Liebe!
